### ISSUE TITLE
Unable to find template "IvoryCKEditorBundle:Form:ckeditor_widget.html.twig" [AGAIN]

### ISSUE THREAD

**Issue by ymihaylov** [egeloen/IvoryCKEditorBundle#346]
Hello there! Im new with Symfony, but I think that is not normal. I wanted to build a (super) Simple admin panel with [EasyAdminBundle](https://symfony.com/doc/master/bundles/EasyAdminBundle/index.html) and I need to use wysiwyg. I found [this article](https://symfony.com/doc/master/bundles/EasyAdminBundle/integration/ivoryckeditorbundle.html) in the docs. I followed the instructions (except ckeditor:install - this command does not work...) and I installed the IvoryCKEditorBundle, but there's a problem with path as its mentioned in [this](https://github.com/egeloen/IvoryCKEditorBundle/issues/292) issue. Im using Symfony 3.4.4. My question is, how can I fix this problem? - changing the ResourceCompilePass is not solution for me.

**Comment by xabbuh** [comment]
The fix proposed by @pixocode in https://github.com/egeloen/IvoryCKEditorBundle/issues/292#issuecomment-304622193 indeed is the way to solve it.

**Comment by ymihaylov** [comment]
Yeah, I saw this, but I think that changing something in vendor directory is not fix. Vendor directory its .gitignored and I cant commit it. Changing someone else code is not good idea.

**Comment by xabbuh** [comment]
Oh well, I meant you could submit a PR to this repo making that change. :)

**Comment by ymihaylov** [comment]
Ok, I will :)

**Comment by Zilus** [comment]
Is the PR ready? 
I have the same issue.
Symfony 3.4 

```Unable to find template "IvoryCKEditorBundle:Form:ckeditor_widget.html.twig" (looked into: project/app/Resources/views, project/vendor/symfony/symfony/src/Symfony/Bridge/Twig/Resources/views/Form).```

**Comment by ymihaylov** [comment]
Actually, no. The problem was that an old version was being installation - version 4.0.6. If you specify the  6.0.1 it works, but not exactly. I just create my own admin and install this bundle to show ckeditor...
